src/util.js

```diff
@@ -271,7 +271,7 @@ function timeif(bool, name, thunk) {
 }
 
 function pipeline(fns) {
-  return _.flowRight.apply(null, fns.reverse());
+  return _.flow.apply(null, fns);
```

Comment:
I added `pipeline` because there wasn't an equivalent in underscore, but yeah, sounds like we can get rid of it now. (At some point.)